swarm.json

```diff
@@ -0,0 +1,53 @@
+{
+  "app_name": "mysql-archiver-test",
+  "services": [
+    {
+      "service_name": "db",
+      "components": [
+        {
+          "component_name": "mysql",
+          "image": "mysql:5.5",
+          "ports": ["3306"],
+          "env": {
+            "MYSQL_ROOT_PASSWORD": "$mysqlpasswd",
+            "MYSQL_DATABASE": "$mysqldb"
+          },
+          "volumes": [
+            {
+              "path": "/var/lib/mysql",
+              "size": "2 GB"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "service_name": "archiver",
+      "components": [
+        {
+          "component_name": "mysql-archiver",
+          "image": "registry.giantswarm.io/yourusername/mysql-archiver:latest",
+          "env": {
+            "DB_NAME": "$mysqldb",
```

Comment:
That would require to change the guide content. Just having it in the repository without explanation wouldn't do, I guess. Maybe I add this later.